index.html

```diff
@@ -0,0 +1,203 @@
+                        Hey there. I’m Nessrine.
+                        <br>
+                        <span>I’m a software developer</span> 
+                    </h2>
+                    <p class="supporting-text">
+                        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        don’t hesitate to contact me.
+                    </p>
+                    <div class="social-media">
+                        <ul>
+                            <li> <a href="#"><img src="images/social-media/github.svg" alt="github-icon"></a></li>
+                            <li><a href="#"><img src="images/social-media/Linkedin icon.svg" alt="linkedin-icon"></a></li>
+                            <li><a href="#"><img src="images/social-media/angellist.svg" alt="angelist-icon"></a></li>
+                            <li><a href="#"><img src="images/social-media/twitter.svg" alt="twitter-icon"></a></li>
+                            <li><a href="#"><img src="images/social-media/medium.svg" alt="medium-icon"></a></li>
+                        </ul>     
+            </div>
+          </div>
+
+        </section>
+
+
+        <section class="work-section">
+            
+            <div class="my-recent-work ">
+                <div class="normal-button"><img src="images/Disabled.svg" alt=""></div>
+                <div class="my-work"> <h2 >My Recent Works</h2></div>
+                <div class="line"></div>
+            </div>
+           
+            <div class="grid-container flex">
+              <div class="card container">
+                <div class="card-work">
+                  <div class="project-information">
+
+                      <p>Multi-Post Stories Gain+Glory</p>
+                      <ul class="flex-list">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                      </ul>
+                      <a href="#"><button type="button">See project</button></a>
+                  </div>
+                </div>    
+              </div>
+              <div class="card container">
+                <div class="card-work">
+                  <div class="project-information">
+  
+                      <p>Multi-Post Stories Gain+Glory</p>
+                      <ul >
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                      </ul>
+                      <a href="#"><button type="button">See project</button></a>
+                  </div>
+                </div>    
+              </div>
+              <div class="card container">
+                <div class="card-work">
+                  <div class="project-information">
+  
```

Comment:
- [ ] I would kindly suggest you to implement the button interaction on each see project button ,The button (e.g., "See project") is a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed (see design)